apps/test/unit/templates/manageStudents/manageStudentsReduxTest.js

```diff
@@ -30,91 +31,10 @@ import manageStudents, {
   transferStudentsFailure
 } from '@cdo/apps/templates/manageStudents/manageStudentsRedux';
 
-const studentEmailData = {
-  1: {
-    id: 1,
-    name: 'StudentNameA',
-    username: 'student1',
-    userType: 'student',
-    age: 17,
-    gender: 'f',
-    loginType: 'email',
-    secretWords: 'wizard',
-    secretPictureName: 'wizard',
-    secretPicturePath: '/wizard.jpg',
-    sectionId: 53,
-    sharingDisabled: true
-  },
-  2: {
-    id: 2,
-    name: 'StudentNameC',
-    username: 'student2',
-    userType: 'student',
-    age: 14,
-    gender: 'm',
-    loginType: 'email',
-    secretWords: 'wizard',
-    secretPictureName: 'wizard',
-    secretPicturePath: '/wizard.jpg',
-    sectionId: 53,
-    sharingDisabled: true
-  },
-  3: {
-    id: 3,
-    name: 'StudentNameD',
-    username: 'student3',
-    userType: 'student',
-    age: 9,
-    gender: 'm',
-    loginType: 'email',
-    secretWords: 'wizard',
-    secretPictureName: 'wizard',
-    secretPicturePath: '/wizard.jpg',
-    sectionId: 53,
-    sharingDisabled: true
-  }
-};
-
-const studentPictureData = {
-  1: {
-    id: 1,
-    name: 'StudentNameA',
-    username: 'student1',
-    userType: 'student',
-    age: 17,
-    gender: 'f',
-    loginType: 'picture',
-    secretWords: 'wizard',
-    secretPictureName: 'wizard',
-    secretPicturePath: '/wizard.jpg',
-    sectionId: 53
-  },
-  2: {
-    id: 2,
-    name: 'StudentNameC',
-    username: 'student2',
-    userType: 'student',
-    age: 14,
-    gender: 'm',
-    loginType: 'picture',
-    secretWords: 'wizard',
-    secretPictureName: 'wizard',
-    secretPicturePath: '/wizard.jpg',
-    sectionId: 53
-  },
-  3: {
-    id: 3,
-    name: 'StudentNameD',
-    username: 'student3',
-    userType: 'student',
-    age: 9,
-    gender: 'm',
-    loginType: 'picture',
-    secretWords: 'wizard',
-    secretPictureName: 'wizard',
-    secretPicturePath: '/wizard.jpg',
-    sectionId: 53
-  }
+const sectionLoginData = {
+  1: sectionLoginFactory.build({id: 1, name: 'StudentNameA', sectionId: 53}),
+  2: sectionLoginFactory.build({id: 2, name: 'StudentNameC', sectionId: 53}),
+  3: sectionLoginFactory.build({id: 3, name: 'StudentNameD', sectionId: 53})
```

Comment:
note that even though `sectionId` is not defined as an `attr` in the `sectionLoginFactory`, we can still include it when we build an instance of our factory object.